InterfaceLib/FakeResources.c

```diff
@@ -465,22 +467,25 @@ void	FakeUpdateResFile( int16_t inFileRefNum )
 	uint32_t		resMapLength = 0;
 	FakeFSeek( currMap->fileDescriptor, resMapOffset, SEEK_SET );
 	
-	FakeFSeek( currMap->fileDescriptor, 4 + 4 + 4 + 4 + 4 + 2, SEEK_CUR );
-	resMapLength += 4 + 4 + 4 + 4 + 4 + 2;
+	resMapLength += (4 + 4 + 4 + 4) + 4 + 2;	// reserved: copy of resource header, next resource handle, file ref
```

Comment:
Do the parentheses add anything here?

Reply:
was just trying to group them as the copy of the resource header